Resources/Private/Language/ExtConf.xlf

```diff
@@ -1,25 +1,29 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes" ?>
 <xliff version="1.0">
-  <file source-language="en" datatype="plaintext" original="messages" date="2017-05-05T10:23:00Z" product-name="service_bw2">
-    <header/>
-    <body>
+	<file source-language="en" datatype="plaintext" original="messages" date="2017-05-05T10:23:00Z"
+				product-name="service_bw2">
```

Comment:
what a huge indent

Reply:
Just looks huge in GitHub. There are 4 tabs to be even with source-language.
![idents](https://user-images.githubusercontent.com/1337769/40119833-f99372fe-591d-11e8-9d71-a063b8516aa4.png)